test/emqtt_kerberos_auth_SUITE.erl

```diff
@@ -0,0 +1,192 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2024 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+%% To run this test suite.
+%% You will need:
+%% - EMQX running with kerberos auth enabled, serving MQTT at port 3883
+%% - KDC is up and running.
+%%
+%% Set up test environment from EMQX CI docker-compose files:
+%% - Update .ci/docker-compose-file/docker-compose.yaml to make sure erlang container is exposing port 3883:1883
+%% - Command to start KDC: docker-compose -f ./.ci/docker-compose-file/docker-compose.yaml -f ./.ci/docker-compose-file/docker-compose-kdc.yaml up -d
+%% - Run EMQX in the container 'erlang.emqx.net'
+%% - Configure EMQX default tcp listener with kerberos auth enabled.
+%% - Copy client keytab file '/var/lib/secret/krb_authn_cli.keytab' from container 'kdc.emqx.net' to `/tmp`
+%%
+-module(emqtt_kerberos_auth_SUITE).
+
+-compile(nowarn_export_all).
+-compile(export_all).
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("common_test/include/ct.hrl").
+-include("emqtt.hrl").
+
+%%------------------------------------------------------------------------------
+%% CT boilerplate
+%%------------------------------------------------------------------------------
+
+all() ->
+    emqtt_test_lib:all(?MODULE).
+
+init_per_suite(Config) ->
+    Host = os:getenv("EMQX_HOST", "localhost"),
+    Port = list_to_integer(os:getenv("EMQX_PORT", "1883")),
```

Comment:
Nit: the module doc above mentions using port 3883, but the default here is 1883.

Reply:
fixed.